llpc/CMakeLists.txt

```diff
@@ -162,6 +162,7 @@ target_include_directories(llpc
         ${PROJECT_SOURCE_DIR}/translator/lib/SPIRV
         ${PROJECT_SOURCE_DIR}/translator/lib/SPIRV/libSPIRV
         ${PROJECT_SOURCE_DIR}/util
+        ${PROJECT_SOURCE_DIR}/../lgc/include
```

Comment:
We should check with Tim.  We may want to place the new header file in "lgc/interface/lgc" with `ElfLinker.h`.  I don't have any strong feelings about this, but this change make me wonder what the original design was suppose to be.

Reply:
You're right Steven; only code inside LGC should access lgc/include. The LLPC front-end, which is the client of LGC, can only access lgc/interface.

The new API into LGC doesn't really fit with the rest of the LGC API, in that it does not require an LgcContext. So just having a whole new header file in lgc/interface/lgc seems fine. (But see my other comment suggesting the interface should be different.)

Reply:
done